tfjs-core/karma.conf.js

```diff
@@ -108,9 +108,11 @@ module.exports = function(config) {
       username: process.env.BROWSERSTACK_USERNAME,
       accessKey: process.env.BROWSERSTACK_KEY
     },
-    captureTimeout: 120000,
+    captureTimeout: 3e5,
```

Comment:
what is this number translate to? will the large number cause the test to stuck and occupy the browserstack instance?

Reply:
300000 ms = 5 min

Reply:
We can change back if this doesn't help, and it's not significantly larger than what we currently have, i.e. 2 min.

Reply:
got ot. sgtm

Reply:
I put it in PR description, this set of number is recommended by Browserstack support.